libs/components/colorpicker/src/lib/modules/colorpicker/types/colorpicker-cmyk.ts

```diff
@@ -1,22 +1,22 @@
 // spell-checker:ignore cmyk, colorpicker
 /**
- * Specifies colors as a combination of cyan, magenta, yellow, and black.
+ * Colors as a combination of cyan, magenta, yellow, and black.
```

Comment:
```suggestion
 * Colors specified as a combination of cyan, magenta, yellow, and black.
```
I wonder if we should stick with `Specifies colors as...` here because this is a description for an interface, not a property. It appears as a standalone sentence under a header, not as one of a series of entries in a table, and it seems like we might want the verb here to provide some clarity. I think dropping the veb works better for table entries that users are scanning than for the descriptions like this that lead into the tables. I'm not sure if this was discussed when we decided to make this change because I only recall the discussion around changing the descriptions for Boolean properties from `Indicates whether...` to just `Whether...` But FWIW, I think we should consider keeping the verbs in instances like this.